### ISSUE TITLE
automatic_releases: too strict interpretation of semantic versioning

### ISSUE THREAD

**Issue by f-laurens** [marvinpinto/actions#199]
The automatic_releases action seems to have a too strict interpretation of semantic versioning:
when `automatic_release_tags` is not set,

- it accepts tags of the form `vX.Y.Z,`
- but rejects tags of the form `flX.Y.Z` (for example) as not _appearing to conform to semantic versioning_

Rejecting tags with a multiple letter prefix seems a bit too strict, since the prefix is **not** part of the semantic version, as stated in the [Static versioning document](https://semver.org/#is-v123-a-semantic-version) itself.

**Comment by lkarlslund** [comment]
An option to also support something that is not semantic versioning would be nice (use the tag no matter what it is)

**Comment by Uzay-G** [comment]
Yeah this would be nice, currently failing on release candidates for example (vX.YrcN), the format proposed by PyPi.

**Comment by sweetlilmre** [comment]
I've also had to work around this issue. As it seems the semVer library is used, I would suggest adding adding an option that would allow the **coerce()** function to be applied to the version. This would seem to be a potentially good middle road.
Either that or an option to ignore the semver requirement. Best would be:

semVerCompliance
- strict
- coerced
- off
